melodic/distribution.yaml

```diff
@@ -677,6 +677,22 @@ repositories:
       url: https://github.com/locusrobotics/catkin_virtualenv.git
       version: master
     status: maintained
+  cis_camera:
+    doc:
+      type: git
+      url: https://github.com/tork-a/cis_camera.git
+      version: 0.0.2
+    release:
+      tags:
+        release: release/melodic/{package}/{version}
+      url: https://github.com/tork-a/cis_camera-release.git
+      version: 0.0.2-1
+    source:
+      test_pull_requests: true
+      type: git
+      url: https://github.com/tork-a/cis_camera.git
+      version: 0.0.2
```

Comment:
A tag is not valid for a source entry, which is why CI is failing.  Please change to a branch.

Reply:
Thank you for pointing out. I changed the wrong tags to the branch.